x-pack/legacy/plugins/monitoring/public/components/chart/_chart.scss

```diff
@@ -1,7 +1,7 @@
-@mixin monitoringNoUserSelect(){
+@mixin monitoringNoUserSelect {
   user-select: none;
-  -webkit-touch-callout: none;
-  -webkit-tap-highlight-color: transparent;
+  -webkit-touch-callout: none; // sass-lint:disable-line no-vendor-prefixes
+  -webkit-tap-highlight-color: transparent; // sass-lint:disable-line no-vendor-prefixes
```

Comment:
These vendor prefixes are so specific they're likely unnecessary. I think they only work on iOS safari.

https://caniuse.com/#search=touch-callout

Reply:
I'm not familiar enough with how the charts work (or should behave) to be comfortable deleting these two lines.

Reply:
These two are definitely safe to delete. They are non-standard. In fact, we are urged **not** to use it in production.
<img width="1110" alt="Screen Shot 2019-11-27 at 12 01 55 PM" src="https://user-images.githubusercontent.com/549577/69744258-de4c2280-110d-11ea-80f3-64a24aa3c69c.png">